zone/command.cpp

```diff
@@ -133,6 +131,7 @@ int command_init(void)
 		command_add("emptyinventory", "Clears your or your target's entire inventory (Equipment, General, Bank, and Shared Bank)", AccountStatus::GMImpossible, command_emptyinventory) ||
 		command_add("enablerecipe", "[Recipe ID] - Enables a Recipe", AccountStatus::QuestTroupe, command_enablerecipe) ||
 		command_add("endurance", "Restores your or your target's endurance.", AccountStatus::Guide, command_endurance) ||
+		command_add("entityvariable", "[Clear|Delete|Set|View] - Modify entity variables for yourself or your target", AccountStatus::GMAdmin, command_entityvariable) ||
```

Comment:
These should probably be lowercase 

`[clear|delete|set|view]`

Reply:
Sounds good.